ListDetail/src/main/java/com/microsoft/device/display/samples/listdetail/ui/view/DetailView.kt

```diff
@@ -148,8 +153,7 @@ fun CameraInfoTile() {
             text = stringResource(R.string.camera),
             style = TextStyle(
                 fontSize = 13.sp,
-                fontWeight = FontWeight.Bold,
-                color = Color.White
```

Comment:
This is similar to the top bar, I think it's good idea to let Compose figure out the correct color - to make that work in light mode, though, I think `onPrimary` and `onBackground` need to be set to white in `Theme.kt`